src/Symfony/Component/Validator/Tests/Constraints/TimezoneValidatorTest.php

```diff
@@ -60,7 +60,25 @@ public function testValidTimezones(string $timezone)
 
     public function getValidTimezones(): iterable
     {
+        // ICU standard (alias/BC in PHP)
+        yield ['Etc/UTC'];
+        yield ['Etc/GMT'];
+        yield ['America/Buenos_Aires'];
```

Comment:
this shows we need #31262 as well, to allow it when using PER_COUNTRY zone (https://3v4l.org/g7Kb7)

status: needs work